### PR TITLE
fix: use `globalThis` rather than `global`

### PR DESCRIPTION
**Pull request by jeswr**
`global` doesn't exist in browser environments.

`globalThis` refers to `global` in node and `window` in browser.

cc @constraintAutomaton

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ module.exports = {
     Headers: false,
     Request: false,
     XMLHttpRequest: false,
+    globalThis: false,
```
Comment (rubensworks) [inline .eslintrc.js:36]: Wondering if we should/could add a linter rule disallowing the use of `global`?
(I did not find a rule for this after a quick search)

Comment (jeswr) [reply .eslintrc.js:36]: I wonder if setting global: true here would work.

Comment (jeswr) [reply .eslintrc.js:36]: Will try soon

Comment (jeswr) [reply .eslintrc.js:37]: Setting them to "off" works (see https://eslint.org/docs/latest/use/configure/language-options#using-configuration-files-1)

**Comment by rubensworks** [comment]
Thanks!

**Pull request merged**